modules/cluster-logging-about-crd.adoc

```diff
@@ -43,10 +43,11 @@ spec:
         request:
           memory: 8G
       proxy:
-        limits:
-          memory: 100Mi
-        requests:
-          memory: 100Mi
+        resources:
```

Comment:
Any reason using 100Mi? How about use the default values?
          limits:
            memory: 256Mi
          requests:
            memory: 256Mi